test/test_csv2cve.py

```diff
@@ -28,8 +28,8 @@ def test_bad_vendor(self):
         self.assertEqual(-2, output)
 
     def test_bad_package(self):
-        """ Test a csv file with a bad package column (should fail) """
-        output = csv2cve(os.path.join(self.CSV_PATH, "bad_package.csv"))
+        """ Test a csv file with a bad product column (should fail) """
+        output = csv2cve(os.path.join(self.CSV_PATH, "bad_product.csv"))
```

Comment:
CI is failing because you didn't also rename this file.  Looking good otherwise, though!